virttest/utils_test/libvirt.py

```diff
@@ -3431,7 +3431,7 @@ def create_secret(params, remote_args=None):
     return sec_uuid
 
 
-def modify_vm_iface(vm_name, oper, iface_dict, index=0):
+def modify_vm_iface(vm_name, oper, iface_dict, index=0, virsh_instance=virsh):
```

Comment:
The default virsh_instance should be base.virsh?

Reply:
Agree, to align with old code

Reply:
According to the discuss, the virsh also can be used.

Reply:
@chloerh both base.virsh and virsh work well.
I decided to use 'virsh' because I think it's more visible than virttest.libvirt_xml.base.virsh.
Would you mind explaining why we use 'base.virsh'?